### PR TITLE
WIP: Bookmark manager style update

### PR DESCRIPTION
**Pull request by bsclifton**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).

Fixes brave#4562
Fixes brave#1994
Fixes brave#4523

## Completely updated styles
Almost done- needs some tweaks
screen shot 2016-10-11 at 2 02 04 am

## Test Plan:
TBD

### CODE REVIEW

**Comment by bsclifton** [comment]
Almost there- one more session and I can knock it out. Basically, all the code is done, just needs style updates.

Since it requires additional work, we can always kick to 0.12.6

File: docs/windowActions.md [hunk 1]
```diff
@@ -891,7 +891,7 @@ See `eventStore.js` for an example use-case
 
 
 
-### onMouseOverBookmarkFolder(folderId, bookmarks, bookmark, xPos, yPos) 
+### setBookmarksToolbarSelectedFolderId(folderId)
```
Comment (bbondy) [inline docs/windowActions.md]: Just noting I think we're trying to move away from setter actions

**Comment by bbondy** [comment]
++

**Comment by bradleyrichter** [comment]
Be sure to ref #4562 for a few more changes...

File: js/components/sortableTable.js [hunk 2]
```diff
@@ -27,7 +27,13 @@ class SortableTable extends ImmutableComponent {
...
   }
   onClick (e) {
-    const targetElement = e.target.parentNode
+    // Work backwards until element is TR
+    let targetElement = e.target
+    while (targetElement) {
+      if (targetElement.tagName === 'TR') break
+      targetElement = targetElement.parentNode
+    }
+    if (!targetElement) return
```
Comment (darkdh) [inline js/components/sortableTable.js]: Nice for parent traversing!!!

File: js/components/sortableTable.js [hunk 3]
```diff
@@ -81,7 +87,12 @@ class SortableTable extends ImmutableComponent {
         ? (typeof this.props.totalRowObjects[parseInt(tableID)][index].toJS === 'function'
           ? this.props.totalRowObjects[parseInt(tableID)][index].toJS()
           : this.props.totalRowObjects[parseInt(tableID)][index])
-        : null
+        : (this.props.rowObjects.size > 0 || this.props.rowObjects.length > 0)
```
Comment (darkdh) [inline js/components/sortableTable.js]: Thanks for taking care of non multi table selection. 👍

**Comment by bsclifton** [comment]
Tests were added here:
https://github.com/brave/browser-laptop/pull/4690

**Pull request merged**